lib/config.js

```diff
@@ -178,7 +180,7 @@ Config.prototype.buildArgs = function () {
     }
 
     // TODO(fixme)
-    args.brave_rewards_enabled = false
+    args.brave_rewards_enabled = true
```

Comment:
actually you should just remove this line

Reply:
it's an override for `brave_rewards_enabled = is_mac || is_linux || is_win || is_android`

Reply:
done